.github/workflows/unstale.yml

```diff
@@ -0,0 +1,41 @@
+name: "rails-bot: Unstale Issues"
```

Comment:
Why do we need t. is action. Doesn't the action/stale already do that?

The README says:

```
If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart
```

Reply:
Because that job only runs once per day (`cron: '0 0 * * *'`), so someone could comment or other action to bump the issue and would be a delay between their action and when the `stale` label was removed.

Just felt the current behavior of instant feedback when you unstale an issue is nice UX, but can remove it if we're fine with possible ~24 hour delay.

Or we can just increase the frequency of the stale job.